src/dials/algorithms/symmetry/cosym/__init__.py

```diff
@@ -265,65 +265,74 @@ def _intialise_target(self):
             nproc=self.params.nproc,
         )
 
-    def _determine_dimensions(self):
-        if self.params.dimensions is Auto and self.target.dim == 2:
-            self.params.dimensions = 2
```

Comment:
We are now not setting this value back into the params object, but I don't think this is used after this point so that is ok?

Reply:
I think so too? :)